### PR TITLE
Update client.read()

### PR DESCRIPTION
**Pull request by pamelafox**
I may be misreading it, but my guess is that it can't both put the data into `buffer` and report num read into `buffer`. So I think its meant to say `bytesRead`. Alternatively, it should probably just not say 'into...' at all, since variable name depends on usage.

### CODE REVIEW

**Comment by bkobkobko** [comment]
Hi @pamelafox 

Your change is not correct.  The return value is indeed the number of bytes actually read, but they are put into the array called buffer (in C/C++ array arguments can be changed inside of functions).  So the text as written before was correct, the bytes are read into the array called buffer.

Hope this makes sense.

**Comment by pamelafox** [comment]
Ohh! I was confused by the ambiguity of the sentence wording. I read it as "it returns the number (...) into buffer". I'd suggest rewording to "Returns the number of bytes (or characters) read." I suppose you can leave it as is if you're trying to make sure people realize which usage you're referring to.

**Comment by kennethlimcp** [comment]
Closing since it has been explained by @bkobkobko!
